src/plugin/cursor.js

```diff
@@ -13,6 +13,9 @@
  * @property {boolean} showTime=false Show the time on the cursor.
  * @property {object} customShowTimeStyle An object with custom styles which are
  * applied to the cursor time element.
+ * @property {string} showTimeYPosition='fixed' Use 'relative' to make the time on
```

Comment:
`showTimeYPosition` sounded like a boolean value until I read the description.
Maybe `followCursorY {boolean}`?